src/Cms/LanguageTranslations.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Kirby\Cms;
+
+use Exception;
+use Kirby\Data\Data;
+use Kirby\Filesystem\F;
+
+/**
+ * With helper methods provides get language translations or
+ * loads from custom `translations` root
+ * @since 5.0.0
+ *
+ * @package   Kirby Cms
+ * @author    Ahmet Bora <ahmet@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class LanguageTranslations
+{
+	protected App   $kirby;
+	protected array $data;
+
+	public function __construct(
+		protected Language $language,
+		self|array $translations = []
+	) {
+		$this->kirby = App::instance();
```

Comment:
Do we need to have the app object in an instance prop as it's the global one anyway?